README.md

```diff
@@ -18,7 +18,7 @@ The data already comes in a Symphony MessageML format (as it is generated by our
 
 ### Opportunity Notification
```

Comment:
You might identify that this event has two different formats.

Something like that: "The opportunity notification may have two different formats: XML or JSON"